kbase-extension/static/kbase/js/widgets/appWidgets2/input/dynamicDropdownInput.js

```diff
@@ -48,15 +46,22 @@ define([
             ddOptions = spec.original.dynamic_dropdown_options || {},
             dataSource = ddOptions.data_source || 'ftp_staging',
             model = {
+                displayValue: undefined,
                 value: undefined,
-            },
-            userId = runtime.userId();
+                descriptionFields: new Set(),
```

Comment:
This will hold the fields used in the description template. So if the template is:
```
NCBI Taxon ID: {{ncbi_taxon_id}}: {{scientific_name}}
```
it will have the values `ncbi_taxon_id` and `scientific_name`.